tests/v2/validation/snapshot/snapshot_restore_k8s_upgrade_test.go

```diff
@@ -0,0 +1,77 @@
+//go:build (validation || extended) && !infra.any && !infra.aks && !infra.eks && !infra.gke && !infra.rke2k3s && !cluster.any && !cluster.custom && !cluster.nodedriver && !sanity && !stress
```

Comment:
Question Markus so this can run on all cluster types except hosted, correct?

Reply:
Correct, do I have to remove them from the build tag in that case?

Reply:
So the tags should be `//go:build (validation || extended || infra.rke2k3s || infra.rke1 || cluster.any) && !infra.any && !infra.aks && !infra.eks && !infra.gke && !sanity && !stress` but have you tried? Is this infra that different from the other cluster types. Can you make sure?

Reply:
If it can we can simplify the tags even further.

Reply:
Yep that worked, providing jobs offline and then we should be good to go.

Reply:
Sweet, so the tag should be `//go:build (validation || extended || infra.any || cluster.any) && !sanity && !stress` and we're good on my end. Thank you!